src/Head.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+
+import { getWeekdaysNames } from './utils/getWeekdaysNames';
+import { HeadProps } from '../typings/react-day-picker';
+
+export const Head: React.FC<HeadProps> = props => {
+  const { locale, showWeekNumber, dayPickerProps } = props;
+  const { classNames, styles, formatWeekdayName } = dayPickerProps;
+  const weekdayNames = getWeekdaysNames(locale, formatWeekdayName);
+  return (
+    <thead style={styles.head} className={classNames.head}>
```

Comment:
Is there a benefit to using the `style` prop over `className`? Doesn't `style` take precedent over anything applied with `className`? I know this can make it difficult to override styles. Just trying to avoid someone trying to override with `!important` at all costs.

Reply:
Good question! These styles are user defined inline-styles: they come from the optional `styles` prop. Example: https://rdp-v8.netlify.com/docs/styling#using-inline-styles

Reply:
Fair enough. If this is something the user has full control over, seems alright to me